sdk/eventhubs/azure-messaging-eventhubs/test/ut/CMakeLists.txt

```diff
@@ -20,13 +20,14 @@ add_executable (
     checkpoint_store_test.cpp 
     consumer_client_test.cpp    
     event_data_test.cpp
+    eventhubs_admin.cpp
     processor_load_balancer_test.cpp
     processor_test.cpp 
     producer_client_test.cpp
     retry_operation_test.cpp
     round_trip_test.cpp
     test_checkpoint_store.hpp
-)
+ "eventhubs_admin_test.cpp")
```

Comment:
Please insert it alphabetically and without quotes, preserving the previous formatting (indent and `)` on a newline)